neurom/view/view.py

```diff
@@ -186,60 +170,47 @@ def _seg_2d(seg):
 
 
 def soma(sm, plane='xy', new_fig=True, subplot=False, **kwargs):
+    '''Generates a 2d figure of the soma.
 
-    """
-    Generates a 2d figure of the soma.
-
-    Parameters
-    ----------
-    soma: Soma
+    Parameters:
+        soma: Soma
         neurom.Soma object
 
-    Options
-    -------
-    plane: str
-        Accepted values: Any pair of of xyz
-        Default value is 'xy'
-
-    linewidth: float
-        Defines the linewidth of the soma.
-        Default value is 1.2
-
-    alpha: float
-        Defines the transparency of the soma.
-        0.0 transparent through 1.0 opaque.
-        Default value is 0.8.
-
-    treecolor: str or None
-        Defines the color of the soma.
-        If None the default value will be used:
-        Soma : "black".
-        Default value is None.
-
-    new_fig: boolean
-        Defines if the tree will be plotted
-        in the current figure (False)
-        or in a new figure (True)
-        Default value is True.
-
-    subplot: matplotlib subplot value or False
-        If False the default subplot 111 will be used.
-        For any other value a matplotlib subplot
-        will be generated.
-        Default value is False.
-
-    limits: list or boolean
-        List of type: [[xmin, ymin, zmin], [xmax, ymax, zmax]]
-        If False the figure will not be scaled.
-        If True the figure will be scaled according to tree limits.
-        Default value is False.
-
-    Returns
-    --------
-    A 2D matplotlib figure with a soma view, at the selected plane.
-
-    """
-
+    Options:
```

Comment:
This is better, but see http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html. Also, I don't think "Options" will be recognised. These should be in "Parameters", with an indication that they are optional. If in doubt, you can build the docs locally and see how it looks. Anyway, I am OK with merging this and fixing it later.